### PR TITLE
Add bitvector regression tests from proof-new

### PR DESCRIPTION
**Pull request by vleni**

### CODE REVIEW

File: test/regress/cli/CMakeLists.txt [hunk 1]
```diff
@@ -464,6 +464,120 @@ set(regress_0_tests
   regress0/bv/fuzz40.delta01.smtv1.smt2
   regress0/bv/fuzz40.smtv1.smt2
   regress0/bv/fuzz41.smtv1.smt2
+  # Added to fill holes in regression tests
+  regress0/bv/holes/concat-flatten.smt2
```
Comment (ajreynol) [inline test/regress/cli/CMakeLists.txt:468]: These need to be alphabetized. Also there should not be spaces between the benchmarks or comments.

**Review by ajreynol — CHANGES_REQUESTED** [review]

File: test/regress/cli/CMakeLists.txt (hunk 1 above)
Comment (vleni) [reply test/regress/cli/CMakeLists.txt]: fixed

**Comment by ajreynol** [comment]
You'll need to remove `--dump-proofs --proof-format=alethe` from several regressions.
Also, you can add `bvite` to the parser to make the other failing regressions pass.

**Comment by vleni** [comment]
> You'll need to remove `--dump-proofs --proof-format=alethe` from several regressions. Also, you can add `bvite` to the parser to make the other failing regressions pass.

How can I remove this flag for a particular regression test?

**Comment by ajreynol** [comment]
Benchmarks have a `COMMAND-LINE:` as the first line in them, e.g. https://github.com/cvc5/cvc5/pull/10343/files#diff-aa141b9e1630a220c581eaf420db17be2b6104401763efceb336d7e21b073e80R1

You should remove any occurrence of `--dump-proofs` or `--proof-format=alethe` from this line.

**Review by ajreynol — APPROVED** [review]

**Pull request merged**